### PR TITLE
[PLUGIN-1100] description of role property in oracle batch sink markdown

### PR DESCRIPTION
**Pull request by itsankit-google**
[PLUGIN-1100](https://cdap.atlassian.net/browse/PLUGIN-1100)

### CODE REVIEW

File: oracle-plugin/docs/Oracle-batchsink.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ You also can use the macro function ${conn(connection-name)}.
 
 **SID/Service Name:** Oracle connection point (Database name or Service name).
 
+**Role** Login role of the user when connecting to the database.
```
Comment (avirmani) [inline oracle-plugin/docs/Oracle-batchsink.md]: is there an example we can give - like what kind of roles?

**Review by avirmani — APPROVED** [review]

File: oracle-plugin/docs/Oracle-batchsink.md (hunk 1 above)
Comment (itsankit-google) [reply oracle-plugin/docs/Oracle-batchsink.md]: ack, added example of roles.

**Review by avirmani — APPROVED** [review]
lgtm

File: oracle-plugin/docs/Oracle-batchsink.md [hunk 2]
```diff
@@ -30,6 +30,8 @@ You also can use the macro function ${conn(connection-name)}.
 
 **SID/Service Name:** Oracle connection point (Database name or Service name).
 
+**Role** Login role of the user when connecting to the database. For eg, NORMAL, SYSDBA, SYSOPER, etc.
```
Comment (rrielley76) [inline oracle-plugin/docs/Oracle-batchsink.md:33]: @itsankit-google If you want to be consistent with the Oracle connection and Oracle batch source markdown they say "Login role of the user when connecting to the database."

**Pull request merged**